tests/Feature/Api/V1/AchievementSetClaimTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Api\V1;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Carbon;
+use LegacyApp\Community\Enums\ClaimSetType;
+use LegacyApp\Community\Enums\ClaimSpecial;
+use LegacyApp\Community\Enums\ClaimStatus;
+use LegacyApp\Community\Enums\ClaimType;
+use LegacyApp\Community\Models\AchievementSetClaim;
+use LegacyApp\Platform\Models\Game;
+use LegacyApp\Platform\Models\System;
+use Tests\TestCase;
+
+class AchievementSetClaimTest extends TestCase
+{
+    use RefreshDatabase;
+    use BootstrapsApiV1;
+
+    public function testGetActiveClaims(): void
+    {
+        /** @var System $system */
+        $system = System::factory()->create();
+        /** @var Game $game */
+        $game = Game::factory()->create(['ConsoleID' => $system->ID]);
+        AchievementSetClaim::factory()->count(51)->create();
+
+        $response = $this->get($this->apiUrl('GetActiveClaims'))
+            ->assertSuccessful();
+
+        $this->assertCount(51, $response->json());
+    }
+
+    public function testGetUserClaims(): void
+    {
+        // Freeze time
+        Carbon::setTestNow(Carbon::now());
+
+        /** @var System $system */
+        $system = System::factory()->create();
+        /** @var Game $game */
+        $game = Game::factory()->create(['ConsoleID' => $system->ID]);
+        /** @var AchievementSetClaim $claim */
+        $claim = AchievementSetClaim::factory()->create([
+            'GameID' => $game->ID,
+            'User' => $this->user->User,
+        ]);
+
+        $this->get($this->apiUrl('GetUserClaims', ['u' => $this->user->User]))
+            ->assertSuccessful()
+            ->assertJson([
+                [
+                    'ClaimType' => ClaimType::Primary,
+                    'ConsoleName' => $system->Name,
+                    'Created' => $claim->Created->__toString(),
+                    'DoneTime' => $claim->Finished->__toString(),
+                    'Extension' => 0,
+                    'GameID' => $game->ID,
+                    'GameIcon' => '/Images/000001.png',
+                    'GameTitle' => $game->Title,
+                    'ID' => $claim->ID,
+                    'MinutesLeft' => Carbon::now()->diffInMinutes(Carbon::now()->addMonths(3)),
```

Comment:
It seems like this should be `Carbon::now()->diffInMinutes($game->Finished)`, but that errors:
```
  -    'MinutesLeft' => 129599,
  +    'MinutesLeft' => '128160.0',
```

Reply:
Ah, yes. `Carbon::now()->diffInRealMinutes()` does the trick.